src/highdicom/pr/content.py

```diff
@@ -0,0 +1,634 @@
+                if max(anchor_point) > 1.0:
+                    raise ValueError(
+                        'All coordinates in the anchor point must be less '
+                        'than or equal to 1 when using DISPLAY units.'
+                    )
+
+        if (tracking_id is None) != (tracking_uid is None):
+            raise TypeError(
+                'If either "tracking_id" or "tracking_uid" is provided, the '
+                'other must also be provided.'
+            )
+        if tracking_id is not None:
+            self.TrackingID = tracking_id
+            self.TrackingUID = tracking_uid
+
+        if graphic_group is not None:
+            if not isinstance(graphic_group, GraphicGroup):
+                raise TypeError(
+                    'Argument "graphic_group" should be of type '
+                    'highdicom.pr.GraphicGroup.'
+                )
+            self.GraphicGroupID = graphic_group.graphic_group_id
+
+    @property
+    def text_value(self) -> str:
+        """str: unformatted text value"""
+        return self.UnformattedTextValue
+
+    @property
+    def bounding_box(self) -> Union[Tuple[float, float, float, float], None]:
+        """Union[Tuple[float, float, float, float], None]:
+        bounding box in the format [left, top, right, bottom]
+
+        """
+        if not hasattr(self, 'BoundingBoxTopLeftHandCorner'):
+            return None
+        return tuple(self.BoundingBoxTopLeftHandCorner) + tuple(
+            self.BoundingBoxBottomRightHandCorner
+        )
+
+    @property
+    def anchor_point(self) -> Union[Tuple[float, float], None]:
+        """Union[Tuple[float, float], None]:
+        anchor point as a (Row, Column) pair of image coordinates
+
+        """
+        if not hasattr(self, 'AnchorPoint'):
+            return None
+        return tuple(self.AnchorPoint)
+
+    @property
+    def units(self) -> AnnotationUnitsValues:
+        """highdicom.pr.AnnotationUnitsValues: annotation units"""
+        if hasattr(self, 'BoundingBoxAnnotationUnits'):
+            return AnnotationUnitsValues(self.BoundingBoxAnnotationUnits)
+        return AnnotationUnitsValues(self.AnchorPointAnnotationUnits)
+
+    @property
+    def tracking_id(self) -> Union[str, None]:
+        """Union[str, None]: tracking identifier"""
+        return getattr(self, 'TrackingID', None)
+
+    @property
+    def tracking_uid(self) -> Union[UID, None]:
+        """Union[highdicom.UID, None]: tracking UID"""
+        if hasattr(self, 'TrackingUID'):
+            return UID(self.TrackingUID)
+        return None
+
+    @property
+    def graphic_group_id(self) -> Union[int, None]:
+        """Union[int, None]: The ID of the graphic group, if any."""
+        return getattr(self, 'GraphicGroupID', None)
+
+
+class GraphicAnnotation(Dataset):
+
+    """Dataset describing related graphic and text objects."""
+
+    def __init__(
+        self,
+        referenced_images: Sequence[Dataset],
```

Comment:
How does this relate to the different `graphic_objects`? Are there any checks that the Anchor Point Annotation Units match the dimensions of the total pixel matrix? I would checks to ensure that points with unit `"PIXEL"` do not extend beyond the image Rows and Columns and that points with unit `"MATRIX"` do not extend beyond the image Total Pixel Matrix Rows and Total Pixel Matrix Columns.

Reply:
That is particularly relevant in slide microscopy, where images within a series may have very different dimensions.

Reply:
Added this check in 1cd763c and then fixed it and added more tests for it in dcd2a5d

Reply:
Awesome!